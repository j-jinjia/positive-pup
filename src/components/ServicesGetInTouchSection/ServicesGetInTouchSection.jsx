import "./ServicesGetInTouchSection.scss";
import Button from "../Button/Button";

const ServicesGetInTouchSection = ({ buttonText, link }) => {
  return (
    <div className="services-get-in-touch-section">
      <h3 className="services-get-in-touch-section__header">
        Have a question about any of our courses?
      </h3>
      <p className="services-get-in-touch-section__subtext">
        Please take a look at our FAQ’s to see if it’s already been answered.
        Alternatively, get in touch using the button below.
      </p>
      <Button buttonText={buttonText} isSecondary link={link} isSmallText />
    </div>
  );
};

export default ServicesGetInTouchSection;
