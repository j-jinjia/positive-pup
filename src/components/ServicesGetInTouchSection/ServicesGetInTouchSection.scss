@use "../../styles/variables" as *;

.services-get-in-touch-section {
  &__header {
    font-family: $font-header;
    font-size: $font-size-h3;
    padding-top: 3rem;
    padding-bottom: 1rem;
    line-height: 150%;
    font-weight: 600;
  }

  &__subtext {
    padding-bottom: 2rem;
    color: $colour-font-paragraph;
  }
}
