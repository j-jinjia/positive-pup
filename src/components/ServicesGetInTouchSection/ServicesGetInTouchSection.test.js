import { customRender } from "../../utils/testUtils";
import ServicesGetInTouchSection from "./ServicesGetInTouchSection";

it("should match ServicesGetInTouchSection component snapshot", () => {
  const { container } = customRender(<ServicesGetInTouchSection />);

  expect(container).toMatchSnapshot();
});
