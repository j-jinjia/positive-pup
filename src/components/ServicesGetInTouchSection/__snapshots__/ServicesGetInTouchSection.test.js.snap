// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match ServicesGetInTouchSection component snapshot 1`] = `
<div>
  <div
    class="services-get-in-touch-section"
  >
    <h3
      class="services-get-in-touch-section__header"
    >
      Have a question about any of our courses?
    </h3>
    <p
      class="services-get-in-touch-section__subtext"
    >
      Please take a look at our FAQ’s to see if it’s already been answered. Alternatively, get in touch using the button below.
    </p>
    <button
      class="button button--secondary button--small-text"
    />
  </div>
</div>
`;
